Bottom: 6a73ca670333a34fb112cdcd207bfd88f13e63e3
Top:    7982a08529a8ce8ac729be3681af6d9683e44d70
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2014-01-01 01:06:25 -0500

Refresh of zarlink

---

diff --git a/package/kernel/linux/modules/sound.mk b/package/kernel/linux/modules/sound.mk
index 7dc59d9..445763e 100644
--- a/package/kernel/linux/modules/sound.mk
+++ b/package/kernel/linux/modules/sound.mk
@@ -304,7 +304,7 @@ define KernelPackage/sound-soc-rt5350-le89156
   KCONFIG:=SND_SOC_RT5350_LE89156
   FILES:=\
 	$(LINUX_DIR)/sound/soc/codecs/snd-soc-le89156.ko \
-	$(LINUX_DIR)/sound/soc/codecs/snd-soc-rt5350-le89156.ko
+	$(LINUX_DIR)/sound/soc/ralink/snd-soc-rt5350-le89156.ko
   AUTOLOAD:=$(call AutoLoad,57,snd-soc-rt5350-le89156)
   DEPENDS:=@TARGET_ramips +kmod-sound-soc-rt5350
   $(call AddDepends/sound)
